### DISCUSSION TITLE
Is there an easy way to remove the start time and end time?

### DISCUSSION THREAD

**Discussion by mohan-nishit** [buildix/timex#20] [Q&A] [answered]
Hello, 

The plugin looks spectacular! 
I have a use case around an employee leave calendar, where start and end times are not needed. Is there an easy way to remove this from the plugin? Or is it a part of your future roadmap?

**Comment by mikrosmile** [comment]
Is all day fits your needs? Or can you please elaborate more. What should be in form where there is no need of start / end times? Or you want to configure in from config? So there will not be time fields at all?

**Comment by mohan-nishit** [comment]
The idea is to have some sort of a configuration option that turns off the
start time and end time altogether, and turns all events into all day
events.
The example would be a Leave calendar where leaves are marked for the full
day, instead of a start time and an end time.

**Comment by mikrosmile** [comment]
Am I right you want it by default for all events? Because now, the toggle "is all day" do this

**Comment by mikrosmile** [comment, marked as answer]
Implemented on latest release v1.1.5
